tools/file_packager.py

```diff
@@ -677,10 +677,12 @@ def generate_js(data_target, data_files, metadata):
           % (start / (1024 * 1024)))
 
     create_preloaded = '''
-          Module['FS_createPreloadedFile'](this.name, null, byteArray, true, true,
-            () => Module['removeRunDependency'](`fp ${that.name}`),
-            () => err(`Preloading file ${that.name} failed`),
-            false, true); // canOwn this data in the filesystem, it is a slide into the heap that will never change\n'''
+          try {
+            await Module['FS_preloadFile'](this.name, null, byteArray, true, true, false, true); // canOwn this data in the filesystem, it is a slide into the heap that will never change
```

Comment:
```suggestion
            await Module['FS_preloadFile'](this.name, null, byteArray, true, true, false, true); // canOwn this data in the filesystem, it is a slice into the heap that will never change
```
Pre-existing, but may as well fix it